.docs/user-guide/storage-providers.md

```diff
@@ -675,22 +706,58 @@ The DigitalOcean block storage driver has the following requirements:
 * Valid DigitalOcean [access token](https://goo.gl/iKoAec)
 
 #### Configuration
-The following example illustrates a minimal configuration for the DigitalOcean
+The following is an example with all possible fields configured. For a running
+example see the `Examples` section.
 block storage driver:
 
 ```yaml
 dobs:
   token:  123456
   region: nyc1
+  statusMaxAttempts: 10
+  statusInitialDelay: 100ms
+  statusTimeout: 60s
 ```
 
+##### Configuration notes
+- The `token` contains your DigitalOcean [access token](https://goo.gl/iKoAec)
+- `region` specifies the DigitalOcean region where volumes should be created
+- `statusMaxAttempts` is the number of times the status of a volume will be
+  queried before giving up when waiting on a status change
+- `statusInitialDelay` specifies a time duration used to wait when polling
+  volume status. This duration is used in exponential backoff, such that the
+  first wait will be for this duration, the second for 2x, the third for 3x,
+  etc. The units of the duration must be given (e.g. "100ms" or "1s").
+- `statusTimeout` is a maximum length of time that polling for volume status can
+  occur. This serves as a backstop against a stuck request of malfunctioning API
+  that never returns.
+
+!!! note
```

Comment:
Hi @codenrhoden,

I prefer not to have two note-styled notes back-to-back. Let's either combine them or separate them somehow.